nautobot/dcim/tests/test_views.py

```diff
@@ -1969,3 +1979,26 @@ def setUpTestData(cls):
             "max_utilization": 50,
             "comments": "New comments",
         }
+
+    def test_power_feed_detail(self):
+        self.add_permissions("dcim.view_powerfeed")
+        # Setup base device info
+        manufacturer = Manufacturer.objects.create(name="Manufacturer", slug="manufacturer-1")
+        device_type = DeviceType.objects.create(manufacturer=manufacturer, model="Device Type 1", slug="device-type-1")
+        device_role = DeviceRole.objects.create(name="Device Role", slug="device-role-1")
+        device = Device.objects.create(
+            device_type=device_type,
+            device_role=device_role,
+            name="Device1",
+            site=self.site,
+        )
+
+        powerport = PowerPort.objects.create(device=device, name="Power Port 1")
+
+        powerfeed = self.powerfeeds[0]
+
+        cable = Cable(termination_a=powerport, termination_b=powerfeed, status=Status.objects.get(slug="connected"))
+        cable.save()
```

Comment:
Can you please try changing this to `Cable.objects.create` and ditch the next line and see if it still works? Just for consistency, thanks.

```suggestion
        cable = Cable.objects.create(termination_a=powerport, termination_b=powerfeed, status=Status.objects.get(slug="connected"))
```

Reply:
@jathanism this works as well. I have pushed the update into the repo.